### PR TITLE
rewrote and organized instructions

### PR DESCRIPTION
**Pull request by jedireza**
Depends on #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,131 @@
...
 
-``` bash
-$ export DEP_OPENSSL_INCLUDE=/usr/local/opt/openssl/include/
-$ export OPENSSL_INCLUDE_DIR=/usr/local/Cellar/openssl/1.0.2f/include/
+```bash
+$ cargo run
 ```
 
-This then builds and runs the project locally:
+Alternativly you can build the app without running it via:
```
Comment (michielbdejong) [inline README.md]: Nit: Alternatively

Comment (jedireza) [inline README.md]: Thanks

File: README.md [hunk 2]
```diff
@@ -1,65 +1,131 @@
...
+```
+
+
+## Selenium tests
+
+You'll need to make sure you install the dependencies via:
+
+```bash
+$ npm install
+```
```
Comment (michielbdejong) [inline README.md:84]: You'll also need to install nodejs, and [install and start the selenium server](https://github.com/fxbox/foxbox/blob/49e94cd2446377e5952f61580e7303b6a66531af/.travis.yml#L10-L13) (see also https://github.com/fxbox/foxbox/pull/19 - maybe we can install and start selenium from package.json).

Comment (jedireza) [inline README.md:84]: Having node installed is implied and addressed under the technology heading (above).

> and install and start the selenium server

Hmm... I didn't install that explicitly today. Maybe it was intalled from the node module?

File: README.md [hunk 3]
```diff
@@ -1,65 +1,131 @@
...
 
-## Target
+## Technologies
 
-Target hardware for prototyping is the Raspberry Pi 2. It's ARMv7 with Neon, but building without Neon support should do for now. Target OS is the latest Raspbian which is based on Debian 8.0 Jessie.
+### Rust
 
+We're using Rust for the daemon/server. Currently 1.8.x nightly is required. We
+plan to track v1.8.x into stable and then continue using stable releases from
+then on. As of 2016-02-04:
```
Comment (fabricedesre) [inline README.md]: We have no such plan to stick on 1.8. We may stay on nightlies longer.

Comment (jedireza) [inline README.md]: Fair enough. I opened #32 and only heard from Hub, which sounded liked a decent plan.

It's easy to remove "We plan to track v1.8.x into stable and then continue using stable releases from then on.". Would you like to mention/address our intent on try getting on stable at some point?

Comment (fabricedesre) [inline README.md]: No, that's really not useful at this point. We may have to stay on nightly for a while (look at servo for instance).

Comment (jedireza) [inline README.md]: Ok thanks. PR updated.

**Comment by jedireza** [comment]
Rebased to resolve conflicts. r? @fabricedesre

File: README.md [hunk 4]
```diff
@@ -1,65 +1,130 @@
...
-Target hardware for prototyping is the Raspberry Pi 2. It's ARMv7 with Neon, but building without Neon support should do for now. Target OS is the latest Raspbian which is based on Debian 8.0 Jessie.
+### Rust
 
+We're using Rust for the daemon/server. Currently v1.8.x nightly is required. As
+of 2016-02-04:
 
-## Toolchain
+```bash
+$ rustc -V
+rustc 1.8.0-nightly (dea183aa8 2016-02-02)
```
Comment (fabricedesre) [inline README.md]: That's not the version to use anymore. Current known good rustc revision is fae516277b6da46b6c1cf568765c90fad2f9ae4b

File: README.md [hunk 5]
```diff
@@ -1,65 +1,130 @@
...
-$ brew install openssl
-$ brew install multirust
-$ multirust update
-$ multirust default nightly
+You should fork the main repo and create pull requests against feature branches
+of your fork. If you need some guidance with this see:
+
+ - https://guides.github.com/introduction/flow/
+ - http://scottchacon.com/2011/08/31/github-flow.html
+
```
Comment (fabricedesre) [inline README.md:49]: nit: extra blank line

File: README.md (hunk 4 above)
Comment (jedireza) [inline README.md]: Thanks. This has been updated.

File: README.md (hunk 5 above)
Comment (jedireza) [inline README.md:49]: I leave two blank lines above each h2. I can remove all of these if you'd like. I think it's easier to read when looking at the plain text.

**Pull request merged**